projects/step-frontend/src/lib/modules/execution/components/alt-execution-tree-widget/alt-execution-tree-widget.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
```

Comment:
OnInit doesn't used here
```suggestion
import { Component } from '@angular/core';
```